appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -67,4 +67,18 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 		int num = sequenceNumber++
 		return "$baseTimestamp$num"
 	}
-}
+
+	def assertDateTimeInLimits(dateTimeString, int rangeSeconds = 10, Date comparisonDateTime = null)
```

Comment:
Possible alternative name is `assertDateEquals` with `rangeSeconds` as third parameter (default 0).